docs/architecture/adr-0026-post-hoc-metrics-scripting.md

```diff
@@ -0,0 +1,75 @@
+# ADR 0026 - Post-hoc Metrics Scripting
+
+[ADR 24](./adr-0024-metrics-scripting.md) answered the question of how to
+track metrics for events in Raster Foundry servers. It doesn't answer how to
+track events for requests elsewhere, for example, total logins by user. Since
```

Comment:
This ADR shouldn't just cover requests -- there are other metrics we need to track and aggregate asynchronously as well like storage used by a user. Right now, that process is handled asynchronously by using S3 storage reports and inserting them into a database table for querying.